corehq/messaging/scheduling/static/scheduling/js/broadcasts_list.js

```diff
@@ -27,9 +27,14 @@ hqDefine("scheduling/js/broadcasts_list", function() {
             "columnDefs": [
                 {
                     "targets": [0],
-                    "render": function() {
-                        // TODO construct this from ID
-                        return 'Delete button';
+                    "className": "text-center",
+                    "render": function(data, type, row) {
+                        var id = row[row.length - 1];
+                        var button_id = 'delete-button-for-scheduled-broadcast-' + id;
+                        return '<button id="' + button_id + '" \
+                                        class="btn btn-danger" \
+                                        onclick="hqImport(\'scheduling/js/broadcasts_list\').deleteScheduledBroadcast(' + id + ')"> \
+                                <i class="fa fa-remove" aria-hidden="true"></i></button>';
```

Comment:
Is this `aria-hidden` intentional?

Reply:
i included because i had seen it in the font awesome example, but agree it probably shouldn't be there for this case https://github.com/dimagi/commcare-hq/pull/19027/commits/043f690774553b3536d300582177568d87daf14f